faker/providers/address/de_AT/__init__.py

```diff
@@ -0,0 +1,155 @@
+        'Jersey', 'Jordanien', 'Kaimaninseln', 'Kambodscha', 'Kamerun',
+        'Kanada', 'Kap Verde', 'Kasachstan', 'Katar', 'Kenia', 'Kirgisistan',
+        'Kiribati', 'Kokosinseln', 'Kolumbien', 'Komoren', 'Kongo', 'Kroatien',
+        'Kuba', 'Kuwait', 'Laos', 'Lesotho', 'Lettland', 'Libanon', 'Liberia',
+        'Libyen', 'Liechtenstein', 'Litauen', 'Luxemburg', 'Madagaskar',
+        'Malawi', 'Malaysia', 'Malediven', 'Mali', 'Malta', 'Marokko',
+        'Marshallinseln', 'Martinique', 'Mauretanien', 'Mauritius', 'Mayotte',
+        'Mazedonien', 'Mexiko', 'Mikronesien', 'Monaco', 'Mongolei',
+        'Montenegro', 'Montserrat', 'Mosambik', 'Myanmar', 'Namibia', 'Nauru',
+        'Nepal', 'Neukaledonien', 'Neuseeland', 'Nicaragua', 'Niederlande',
+        'Niederländische Antillen', 'Niger', 'Nigeria', 'Niue', 'Norfolkinsel',
+        'Norwegen', 'Nördliche Marianen', 'Oman', 'Osttimor', 'Pakistan',
+        'Palau', 'Palästinensische Gebiete', 'Panama', 'Papua-Neuguinea',
+        'Paraguay', 'Peru', 'Philippinen', 'Pitcairn', 'Polen', 'Portugal',
+        'Puerto Rico', 'Republik Korea', 'Republik Moldau', 'Ruanda',
+        'Rumänien', 'Russische Föderation', 'Réunion', 'Salomonen', 'Sambia',
+        'Samoa', 'San Marino', 'Saudi-Arabien', 'Schweden', 'Schweiz',
+        'Senegal', 'Serbien', 'Serbien und Montenegro', 'Seychellen',
+        'Sierra Leone', 'Simbabwe', 'Singapur', 'Slowakei', 'Slowenien',
+        'Somalia', 'Sonderverwaltungszone Hongkong',
+        'Sonderverwaltungszone Macao', 'Spanien', 'Sri Lanka',
+        'St. Barthélemy', 'St. Helena', 'St. Kitts und Nevis', 'St. Lucia',
+        'St. Martin', 'St. Pierre und Miquelon',
+        'St. Vincent und die Grenadinen', 'Sudan', 'Suriname',
+        'Svalbard und Jan Mayen', 'Swasiland', 'Syrien',
+        'São Tomé und Príncipe', 'Südafrika',
+        'Südgeorgien und die Südlichen Sandwichinseln', 'Tadschikistan',
+        'Taiwan', 'Tansania', 'Thailand', 'Togo', 'Tokelau', 'Tonga',
+        'Trinidad und Tobago', 'Tschad', 'Tschechische Republik', 'Tunesien',
+        'Turkmenistan', 'Turks- und Caicosinseln', 'Tuvalu', 'Türkei',
+        'Uganda', 'Ukraine', 'Ungarn', 'Uruguay', 'Usbekistan', 'Vanuatu',
+        'Vatikanstadt', 'Venezuela', 'Vereinigte Arabische Emirate',
+        'Vereinigte Staaten', 'Vereinigtes Königreich', 'Vietnam',
+        'Wallis und Futuna', 'Weihnachtsinsel', 'Westsahara',
+        'Zentralafrikanische Republik', 'Zypern', 'Ägypten',
+        'Äquatorialguinea', 'Äthiopien', 'Äußeres Ozeanien', 'Österreich',
+    )
+
+    def street_suffix_short(self):
+        return self.random_element(self.street_suffixes_short)
+
+    def street_suffix_long(self):
+        return self.random_element(self.street_suffixes_long)
+
+    def city_name(self):
+        return self.random_element(self.cities)
+
+    def state(self):
+        return self.random_element(self.states)
+
+    def country(self):
```

Comment:
we don't need to override this method if it's exactly the same as the superclass